ui/src/app/admin/services/AdminService.js

```diff
@@ -43,11 +43,19 @@
       });
     }
 
+    var visitMinio = function() {
+      var url = 'http://10.61.32.83:9000'
```

Comment:
Minio can run on any server to store data, may not always on `10.61.32.83` and developers can explicitly define that as well. So shall we get a better way to this?

Reply:
that was for testing purposes locally, either we need Minio client SDK to be installed on user's system OR we need a cloud server where Minio can be deployed, I worked all this assuming Bassa will have a cloud server in future, so URL will not have a constant value in that case.

Reply:
I think this require a good discussion actually before going for `k8s`. Add this to discussion as well @demon-36 @kmehant 
@demon-36 for right now, keep it `localhost:9000`. We will discuss better option for sure :)

Reply:
Are we making the Minio file system the default file system for Bassa or use use it based on a configuration?

Reply:
Right now, till Vibhor work get completed, we will keep the Minio as default FS. After all things getting good with Vibhors FS implementation, we can have a decision. What do you say @agentmilindu ?

Reply:
The simplest thing should be the default. It could be writing to a local directory. Everything else should come from configurations. Also, keep in mind primary use case of Bassa is installing on a on-pren server or cluster, downloading to a disk in local network on of peak time. Then anyone can copy through local network at anytime.

Reply:
@agentmilindu  @vivonk   Minio is going to work exactly same way we expect when it will run on server, files will be downloaded to Minio server, when user will try to download completed file to disk then Bassa will fetch files from Minio and will save it to disk.